flink-formats-aws/flink-json-glue-schema-registry/pom.xml

```diff
@@ -61,6 +61,14 @@ under the License.
             <version>${glue.schema.registry.version}</version>
         </dependency>
 
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.30</version>
+            <scope>test</scope>
+        </dependency>
+
```

Comment:
Lombok had been consumed as transitive dependency of `software.amazon.glue:schema-registry-serde`.
Starting from `schema-registry-serde` 1.1.15 it is no longer provided.